com.vagueentertainment.Framed.yml

```diff
@@ -0,0 +1,74 @@
+app-id: com.vagueentertainment.Framed
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: framed
+modules: 
+  
+  - name: framed
+    buildsystem: simple
+    build-commands:
+      - install -D -m755 Framed.x86_64 /app/bin/Framed.x86_64
```

Comment:
does this work on aarch64? if not at all then you need to limit the architectures you build for with a flathub.json file. e.g. https://github.com/flathub/com.slack.Slack/blob/master/flathub.json

Reply:
I'd suggest building from source, but the MIT license seems to be somewhat false advertising as the upstream repo doesn't contain the sources.